### PR TITLE
Handle stale sidekiq queue and process metrics

### PR DESCRIPTION
**Pull request by sosedoff**
This PR adds a few handling bits for dealing with stale sidekiq metics.
Root cause of the issue is described in #261.

### CODE REVIEW

**Comment by sosedoff** [comment]
With new `MetricsContainer` we can define auto-expiry upon insertion and iteration. Default is 60s, with ability to customize the value per collector.

Also, didn't want to touch other collectors, but we could expand the usage of the new class in a separate PR (if necessary).

File: lib/prometheus_exporter/server/metrics_container.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    end
+
+    private
+
+    def get_time
+      ::Process.clock_gettime(::Process::CLOCK_MONOTONIC)
+    end
+
+    def wrap_expire(method_name, &blk)
+      expire.send(method_name, &blk)
```
Comment (SamSaffron) [inline lib/prometheus_exporter/server/metrics_container.rb:53]: minor ... but this is a bit surprising I think, this is a bit clearer. (and more verbose) 

```
def wrap_expire(method_name, &blk)
  expire
  @data.public_send(method_name, &blk)
end
```

**Review by SamSaffron — APPROVED** [review]

**Comment by SamSaffron** [comment]
overall change looks good :confetti_ball:  

would love an extra PR to expand usage.

**Comment by SamSaffron** [comment]
Thanks @sosedoff !

**Pull request merged**